src/Uploadcare/Api.php

```diff
@@ -137,9 +138,18 @@ class Api
      * @param string $cdn_host CDN Host
      * @param string $cdn_protocol CDN Protocol
      * @param integer $retry_throttled Retry throttled requests this number of times
+     * @param int $lifeTime Secure signature expire time in seconds for signed uploads.
```

Comment:
```suggestion
     * @param int $lifetime Secure signature expire time in seconds for signed uploads.
```

lifetime, not life time
here and everywhere